scala/integration/intellij-bsp/src/org/jetbrains/plugins/scala/bsp/config/ScalaBspProjectAwareExtension.scala

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.plugins.scala.bsp.config
+
+import com.intellij.openapi.externalSystem.autoimport.ExternalSystemProjectId
+import com.intellij.openapi.vfs.VirtualFile
+import org.jetbrains.plugins.bsp.config.BspProjectAwareExtension
+
+import java.util.{List => JavaList}
```

Comment:
now in this file it's `JavaList` and in `scala/integration/intellij-bsp/src/org/jetbrains/plugins/scala/bsp/extension/ScalaBuildTargetClassifier.scala` it's `util.List`. You can use `util.List` here as well if you prefer, but please be consistent :) (previously I didn't check what the convention is in this repo, so the best choice is to be consistent with it as well)

Reply:
I think there is no convention in the repo since both methods are used but as simple `util.List` is used more frequently, we will stick to it.